app/connectors/SubmissionConnector.scala

```diff
@@ -17,55 +17,33 @@
 package connectors
 
 import config.AppConfig
+import play.api.Logger
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
 
 import javax.inject.Inject
-import uk.gov.hmrc.http.{Authorization, HeaderCarrier, HeaderNames, HttpClient, HttpResponse}
-
-import java.time.ZonedDateTime
-import java.time.format.DateTimeFormatter
-import java.util.UUID
 import scala.concurrent.{ExecutionContext, Future}
 import scala.xml.NodeSeq
 
 class SubmissionConnector @Inject() (
   val config: AppConfig,
   http: HttpClient
 )(implicit ec: ExecutionContext) {
-
-  val submissionUrl = s"${config.submissionUrl}/dac6/dct06/v1"
+  private val logger: Logger = Logger(this.getClass)
```

Comment:
If you make the class extend Logging (play.api.Logging) you don't need to instantiate a logger. PR approved though, just a suggestion.